src/commands/Minion/cast.ts

```diff
@@ -105,9 +124,9 @@ export default class extends BotCommand {
 		return msg.send(
 			`${msg.author.minionName} is now casting ${quantity}x ${
 				spell.name
-			}, it'll take around ${formatDuration(
-				duration
-			)} to finish. Removed ${cost} from your bank. **${xpHr}**`
+			}, it'll take around ${formatDuration(duration)} to finish. Removed ${cost}${
+				spell.gpCost ? ` and ${gpCost} Coins` : ``
+			} from your bank. ** ${xpHr}**`
```

Comment:
remove space after **
```suggestion
			} from your bank. **${xpHr}**`
```